mscore/scoreaccessibility.cpp

```diff
@@ -163,10 +189,16 @@ void ScoreAccessibility::currentInfoChanged()
             }
       else if (score->selection().isRange()) {
             QString barsAndBeats = "";
-            std::pair<int, float> bar_beat;
-
-            bar_beat = barbeat(score->selection().startSegment());
-            barsAndBeats += " " + tr("Start Measure: %1; Start Beat: %2").arg(QString::number(bar_beat.first)).arg(QString::number(bar_beat.second));
+            std::tuple<int, int, float> bar_beat = bar_beat = barbeat(score->selection().startSegment());
```

Comment:
I'm starting to think now that this may have been intentional but, anyhow...

Reply:
more like a copy/paste error?

Reply:
Thank you, yes, this was a copy and paste error. I've fixed them all now.